src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.iOS.cs

```diff
@@ -61,7 +61,6 @@ public PointerPoint GetCurrentPoint(UIElement relativeTo)
 			touchType switch
 			{				
 				UITouchType.Stylus => PointerDeviceType.Pen,
-				UITouchType.IndirectPointer => PointerDeviceType.Mouse,
```

Comment:
Should this be in this PR?
I think @dr1rrb worked on pointers?

Reply:
This line prevents the solution from building on iOS

Reply:
For stuff like this it's better to create an issue or contact someone on the core team directly, rather than including it in an unrelated PR where it might accidentally get committed and break functionality.

In this case it seems that `UITouchType.IndirectPointer` was added in Xamarin.iOS 13.16: https://docs.microsoft.com/en-us/xamarin/ios/release-notes/api-changes/ios-13-14-1-13-16-0, so it's probably a question of an older version of Visual Studio.